client/js/widgets/button.js

```diff
@@ -53,12 +53,19 @@ export class Button extends Widget {
 
     if(this.p('debug'))
       $('#debugButtonOutput').textContent = '';
-
```

Comment:
Any reason for removing the empty line?

Reply:
No reason, I originally had a lot of logs to see what different variables held, and when I removed them I must have removed the extra line